### ISSUE TITLE
Add support for self-signed certificate and spend_time not showed in issue

### ISSUE THREAD

**Issue by windworship** [kriskbx/gitlab-time-tracker#38]
## tldr
Please add support for self-signed certificate in request options and checkout why spend_time not showed in our environment

## self-signed certificate
We are currently hosting gitlab in our local network, thus https certificate is signed by a local CA. 
And from 1.7.0, we can not generate reports due to the certiciate check.
![image](https://user-images.githubusercontent.com/7194287/34140493-09f8ad30-e4b6-11e7-94ca-a847a387209e.png)

After searching, we add one line in `plumbing.js` at line 35
```js
process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```

and now the check can pass

Cound you please add some options to not check the certificate of ssl in configuration?

## Spend_time missing

There is still an issue when we try to generate the report.

### TIME STATS

* **total estimate**: 1h 30m
* **total spent**: 2d 1h 45m
* **spent**: 2d 1h 45m

* **jimin**: 22m
* **heheqiao**: 2d 1h 23m

### ISSUES

| iid  | title                                    | spent | total estimate |
| ---- | ---------------------------------------- | ----- | -------------- |
| 2    | DayTransactionHistory.query_by_day       |       |                |
| 4    | StockPool.query_stock_ids                |       |                |
| 3    | HoldStockDetailInformation.query_stock_ids |       |                |
| 1    | IndexTransactionHistory.query_by_day     |       |                |

### TIME RECORDS

| user     | date                | type          | iid  | time   |
| -------- | ------------------- | ------------- | ---- | ------ |
| heheqiao | 14.12.2017 16:56:29 | Issue         | 2    | 51m    |
| heheqiao | 14.12.2017 16:56:50 | Issue         | 4    | 41m    |
| heheqiao | 14.12.2017 16:56:41 | Issue         | 3    | 12m    |
| heheqiao | 14.12.2017 16:56:13 | Issue         | 1    | 21m    |

After commenting `/spend` in issue, the `spend_time` is still missing in the `ISSUES`, despite that the time records showed what we've done.

Could you please provide some workaround in this problem?

**Comment by kriskbx** [comment]
Thank you for your suggestion and bug report. I just pushed new changes to the master branch that introduce an `unsecure` config option. Can you pull the master branch and try it out? Unfortunately I'm not able to reproduce your second issue regarding missing times in the issue table. Can you try it again with the `--verbose` flag to see if there are errors or warnings of any kind? If not, is there any way that I could get read-only API access to the project, so I can debug with your actual data?

**Comment by kriskbx** [comment]
I just released version 1.7.4 containing this feature. Make sure to update and feel free to reopen the issue if you still have trouble using it.
